coursemapper-kg/concept-map/src/pipelines/concept_map.py

```
@@ -62,13 +62,13 @@
 
         file_embedding = self.embedding_service.encode(' . '.join(pages_df['page']))
 
-        material_node = Node(material_id, material_name, '', 'material', 1, '', '\n\n'.join(pdf_text), True, file_embedding, [], Config.EMBEDDING_MODEL)
+        material_node = Node(material_id, material_name, '', 'material', 1, '', '\n\n'.join(pdf_text), True, False, False,False, file_embedding, [], Config.EMBEDDING_MODEL)
         graph.add_node(material_node)
 
         for i, (page, page_parts, page_embedding) in enumerate(pages_df.iter_rows()):
             push_log_message(f'Processing page {i+1}/{pages_df.height}')
 
-            page_node = Node(f'{material_id}_slide_{i+1}', f'slide_{i+1}', '', 'Slide', 1, '', page, False, page_embedding)
+            page_node = Node(f'{material_id}_slide_{i+1}', f'slide_{i+1}', '', 'Slide', 1, '', page, False, False, False,False, page_embedding , page_parts, Config.EMBEDDING_MODEL )
```

Ruff (0.12.2)

Boolean positional value in function call

(FBT003)